### ISSUE TITLE
Remove ethernet network device not managed the Ubuntu 16.04 interface

### ISSUE THREAD

**Issue by thyeun** [docker/docker#26686]
OS : Ubuntu 16.04
Docker : v 1.12.1

After using syntax "sudo service docker start" and "sudo docker run hello-world", getting the same issue like the below picture

![junzm](https://cloud.githubusercontent.com/assets/1939446/18616521/0bd14d16-7df0-11e6-8afc-f2ece5e7d332.jpg)

**Comment by justincormack** [comment]
`NetworkManager` does not play well with network interfaces that it does not create, I think that is the issue. I would uninstall it or file an issue with Ubuntu, I don't believe there is anything that Docker can do about this. There are some workarounds here http://support.qacafe.com/knowledge-base/how-do-i-prevent-network-manager-from-controlling-an-interface/ - I usually do the last one of uninstalling `NetworkManager`...

**Comment by thyeun** [comment]
@justincormack haha... any way thanks for your link.

**Comment by thaJeztah** [comment]
Looks like this is not a docker bug; are you ok with closing this one @thyeun?

**Comment by stucki** [comment]
This is indeed not a Docker issue as it also happens with other virtual devices (VirtualBox, etc.) but I'm still adding a link to the upstream issue here, so that it can be found by other users...

https://bugs.launchpad.net/ubuntu/+source/network-manager/+bug/1458322

**Comment by pavelsr** [comment]
Here is step-by-step summary from link:

```
sudo apt-get build-dep network-manager
apt-get source network-manager
cd network-manager-1.2.6
cd debian/patches
wget https://bugs.launchpad.net/ubuntu/+source/network-manager/+bug/1458322/+attachment/4914082/+files/hide_unmanaged_interfaces.patch
cd -
dpkg-buildpackage -b
```

**Comment by thyeun** [comment]
@pavelsr , thank you so much, it look pretty now, by follow your step-by-step.

will this step-by-step "fix" for ubuntu 18.04 (after the launching)?

**Comment by sojab0on** [comment]
I think i can, but the version of network-manager in 18.04 is newer zo the version in de guide needs to be replaced for 18.04 to 1.10.6 instead of 1.2.6 

i am running the dpkg build now as we speak and see what happens
